tests/lib/snaps/store/test-snapd-gpio-chardev/snapcraft.yaml

```diff
@@ -0,0 +1,50 @@
+name: test-snapd-gpio-chardev
+version: "1.0"
+summary: Basic gpio-chardev snap
+description: A basic snap which access to exported gpiod device
+grade: stable
+confinement: strict
+base: core24
+
+apps:
+  cmd:
+    command: bin/cmd
+    plugs: [gpio-chardev-0, gpio-chardev-1]
+  svc:
+    command: bin/service.sh
+    daemon: oneshot
+    plugs: [gpio-chardev-0, gpio-chardev-1]
+
+plugs:
+  gpio-chardev-0:
+    interface: gpio-chardev
+  gpio-chardev-1:
+    interface: gpio-chardev
+
+parts:
+  libgpiod:
+    source: git://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git
+    source-branch: v2.1.x
+    plugin: autotools
+    autotools-configure-parameters:
+      - --prefix=/usr
+      - --enable-tools=yes
+    build-packages:
+      - build-essential
+      - autoconf-archive
+      - autoconf
```

Comment:
Are those needed with `plugin: autotools`?

Reply:
Nice catch, This was copy-pasted from `test-snapd-gpio-control`, building without them worked as expected, Thanks!